extensions/vi-mode/core.lisp

```diff
@@ -3,6 +3,7 @@
         :lem
         :lem/universal-argument)
   (:import-from :cl-package-locks)
+  (:import-from #:cl-ppcre)
```

Comment:
we use keywords for package-designator in this project, so it would be nice to see it unified.